substrate/utils/frame/benchmarking-cli/src/pallet/mod.rs

```diff
@@ -181,6 +181,13 @@ pub struct PalletCmd {
 	#[arg(long, value_enum)]
 	pub genesis_builder: Option<GenesisBuilder>,
 
+	/// The preset that we expect to find in the GenesisBuilder runtime API.
+	///
+	/// This can be useful when a runtime has a dedicated benchmarking preset instead of using the
+	/// default one.
+	#[arg(long, default_value = "development")]
```

Comment:
@michalkucharczyk pointed me to the const: `sp_genesis_builder::DEV_RUNTIME_PRESET`, this module also has dependency `sp_genesis_builder`, but not sure if we can reuse it here

Reply:
Yes works, thanks.